### DISCUSSION TITLE
Number (+/-) input

### DISCUSSION THREAD

**Discussion by dohahelmy** [twbs/bootstrap#32414] [Ideas] [answered]
Hello,
I would like to ask if there is any possibility to add custom **number (+/-) input** to form elements?
something similar to the following image
![image](https://user-images.githubusercontent.com/17557397/101741386-a9213100-3ad2-11eb-8c3f-e26ef17e21c7.png)

**Comment by mdo** [comment, marked as answer]
You can use input groups to accomplish this right now. You'd have to write your own JS to do that, though.

Alternatively you can use the number input. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number
